lib/trie/database.go

```diff
@@ -128,6 +128,26 @@ func (t *Trie) load(db chaindb.Database, curr node) error {
 	return nil
 }
 
+// GetDBKey return hash of each key of the trie.
```

Comment:
maybe rename to `GetNodeHashes` or something more descriptive?

Reply:
Done